scripts/knife_edge/test_ld_zero_conditions.py

```diff
@@ -193,22 +207,64 @@ def get_K(Pi_s, td_policy, lambda_):
     "ld_zero_by_t_projection": 3,
     "ld_zero_by_r_projection": 1,
     "ld_zero_by_wr_projection": 1,
+    "tiger-alt-start": 0,
+    "network": 0,
+    "tmaze_5_two_thirds_up": 0,
+    "example_7": 0,
+    "4x3.95": 0,
+    "cheese.95": 0,
+    "network": 0,
+    "shuttle.95": 0,
+    "paint.95": 0,
+    "hallway": 0,
+    "bridge-repair": 0,
 }
 # %%
 data = []
 ps = np.linspace(0, 1, n_samples)
 for spec, n_params in specs_and_n_params.items():
     pomdp, info = load_pomdp(spec)
-    all_ps = np.reshape(np.meshgrid(*[ps] * n_params), (n_params, -1)).T
+    if n_params > 0:
+        all_ps = np.reshape(np.meshgrid(*[ps] * n_params), (n_params, -1)).T
+    else:  # n_params == 0
+        all_ps = range(10)
     for probs in tqdm(all_ps):
-        pi = get_policy(spec, *probs)
+        if n_params > 0:
+            pi = get_policy(spec, *probs)
+        else:
+            pi = get_policy(spec)
+            assert isinstance(pi, str) and pi == "random"
+            n_o = pomdp.phi.shape[1]
+            n_a = pomdp.T.shape[0]
+            pi = np.random.rand(n_o, n_a)
+            pi = pi / pi.sum(axis=1)[:, None]
```

Comment:
Can we move this into the `get_policy()` function? Oh, I guess it needs a size. Can we make a `get_random_policy(size)` function?

Reply:
Done!